pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers=[
 [tool.poetry.dependencies]
 python=">=3.8.0,<3.12"
 numba=">=0.51.0"
-qibo=">=0.1.15"
+qibo={ git = "https://github.com/qiboteam/qibo.git" , branch = "new_gates" }
```

Comment:
This should be changed to what it was or remove the branch (use qibo master). If it doesn't work with the previous version we could consider making a release in qibo.

Reply:
It should work with `qibo` from `master` now, but not with the latest stable version

Reply:
Just to confirm: the current one is `0.1.15` and the next one is `0.1.16`. So it's `0.1.16`, right?